src/delta.rs

```diff
@@ -252,6 +313,28 @@ fn handle_hunk_line(painter: &mut Painter, line: &str, state: State, config: &Co
     }
 }
 
+fn handle_file_uniqueness(
```

Comment:
When I read the code I struggled a bit to understand this function name. The other analogous functions are called `handle_commit_meta_header_line`, `handle_file_meta_header_line`, `handle_hunk_meta_line`, `handle_hunk_line`.

WDYT about `handle_unified_diff_unique_file_name`? And would you mind adding a brief docstring explaining what it does?

Reply:
Or `handle_directory_diff_unique_file_name`?